script/setup_local_dev.sh

```diff
@@ -0,0 +1,171 @@
+#!/usr/bin/env bash
+
+: "${ENSURE_K3D:="false"}" # will ensure k3d is installed if set to "true"
+: "${ENSURE_KUBECTL:="false"}" # will ensure kubectl is installed if set to "true"
+: "${ENSURE_HELM:="false"}" # will ensure helm is installed if set to "true"
+
+: "${K3D_VERSION:="v4.4.8"}" # version of k3d to install, empty value uses latest one available. currently pinned to latest known working (due to unresolved port mapping issue: https://github.com/k3d-io/k3d/issues/380)
+: "${K3D_INSTALL_URL:="https://raw.githubusercontent.com/rancher/k3d/main/install.sh"}" # URL to k3d installer script
+: "${K3S_VERSION:="v1.22.8-k3s1"}" # version of k3s to run in k3d cluster, empty value uses default specified by k3d install
```

Comment:
Could we use `v.1.21.1`? just make it consistent with demo and sandbox. https://github.com/flyteorg/flyte/blob/b7928c075225843d097a1c598ccd7f51b2e87069/docker/sandbox-lite/Dockerfile#L27

Reply:
Sure thing, updated :+1: Sorry, I must've missed the `sandbox-lite` image used that version when skimming over the Dockerfile